tests/client/core/test_utils.js

```diff
@@ -713,3 +727,104 @@ describe('render404IfConfigKeyIsFalse', () => {
     assert.equal(props.size, 'large');
   });
 });
+
+describe('getCompabilityVersions', () => {
+  it('gets the min and max versions', () => {
+    const addon = {
+      ...fakeAddon,
+      current_version: {
+        ...fakeAddon.current_version,
+        compatibility: {
+          firefox: {
+            max: '20.0.*',
+            min: '11.0.1',
+          },
+        },
+      },
+    };
+    const { maxVersion, minVersion } = getCompabilityVersions({
+      addon, clientApp: 'firefox' });
+
+    assert.equal(maxVersion, '20.0.*');
+    assert.equal(minVersion, '11.0.1');
+  });
+
+  it('gets null if the clientApp does not match', () => {
+    const addon = {
+      ...fakeAddon,
+      current_version: {
+        ...fakeAddon.current_version,
+        compatibility: {
+          firefox: {
+            max: '20.0.*',
+            min: '11.0.1',
+          },
+        },
+      },
+    };
+    const { maxVersion, minVersion } = getCompabilityVersions({
+      addon, clientApp: 'android' });
+
+    assert.equal(maxVersion, null);
+    assert.equal(minVersion, null);
+  });
+
+  it('returns null if clientApp has no compatibility', () => {
+    const addon = {
+      ...fakeAddon,
+      current_version: {
+        ...fakeAddon.current_version,
+        compatibility: {},
+      },
+    };
+    const { maxVersion, minVersion } = getCompabilityVersions({
+      addon, clientApp: 'firefox' });
+
+    assert.equal(maxVersion, null);
+    assert.equal(minVersion, null);
+  });
+
+  it('returns null if current_version does not exist', () => {
+    const addon = {
+      ...fakeAddon,
+      current_version: null,
+    };
+    const { maxVersion, minVersion } = getCompabilityVersions({
+      addon, clientApp: 'firefox' });
+
+    assert.equal(maxVersion, null);
+    assert.equal(minVersion, null);
+  });
+
+  it('returns null if addon is null', () => {
+    const { maxVersion, minVersion } = getCompabilityVersions({
+      addon: null, clientApp: 'firefox' });
+
+    assert.equal(maxVersion, null);
+    assert.equal(minVersion, null);
+  });
+});
+
+describe('isCompatibleWithUserAgent', () => {
+  it('should mark non-Firefox UAs as incompatible', () => {
+    isCompatibleWithUserAgent({ userAgent: userAgents.chrome[0] });
```

Comment:
I don't see any assertions. What is this test doing? It also looks like a dupe of https://github.com/mozilla/addons-frontend/blob/43b3dfd55121535d1acf249153b02161e214225f/tests/client/core/test_utils.js#L193